src/mrack/run.py

```diff
@@ -256,15 +256,15 @@ def handle(*args, **kwargs):
             logger.exception(exc)
             raise exc
 
-        return rc
+        return ret_code
 
     return handle
 
 
 @exception_handler
 def run():
     """Run the app."""
-    mrackcli(obj={})
+    mrackcli()  # pylint: disable=no-value-for-parameter
```

Comment:
Will this work?  AFAIK the `obj={}` is needed for correct work of context.

Reply:
mrack up and destroy works fine for me without the `obj={}`

Reply:
Shouldn't it be `ctx={}` to have some effect?

_(Not actually related interesting "known gotcha": https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments.)_

Reply:
I vaguely remember that it was needed. It is documented here: https://click.palletsprojects.com/en/7.x/commands/#nested-handling-and-contexts But if **all** commands with various options used work the same without it then we can remove it.

Reply:
i reverted this change